### PR TITLE
modified Main.java

### PR DESCRIPTION
**Pull request by ehkhalilov**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -1,9 +1,10 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello World!");
-        Customer customer = new Customer("Elnur", 29);
-        Customer customer3 = new Customer("Aydan",20);
-        Customer customer = new Customer("Elnur", 30,"Xalilov ");
+        Customer customer = new Customer("Elnur", 29, "");
```
Comment (ehkhalilov) [inline src/Main.java:4]: kjasdkajshdkj

**Review by gulayhajiyevaa — APPROVED** [review]

**Review by AminRamazanov — APPROVED** [review]

**Review by aydanrahimova — APPROVED** [review]

**Review by ehkhalilov — COMMENTED** [review]
test comment

**Review by aydanrahimova — CHANGES_REQUESTED** [review]
dkjgfd

**Review by gulayhajiyevaa — CHANGES_REQUESTED** [review]
test

**Pull request merged**